lib/polyfill/media_capabilities.js

```diff
@@ -49,12 +50,10 @@ shaka.polyfill.MediaCapabilities = class {
         shaka.util.Platform.isPS4() ||
         shaka.util.Platform.isWebOS() ||
         shaka.util.Platform.isTizen() ||
-        shaka.util.Platform.isChromecast()) {
+        shaka.util.Platform.isChromecast() ||
+        shaka.util.Platform.isAndroidCastDevice()) {
```

Comment:
isChromecast already includes Android Chromecast, so this line is not needed

Reply:
Oops, thank you! I've simplified it and removed the `isAndroidCastDevice()` method altogether. If we end up needing Android Cast specific logic later, we can reinstate it then.